rdmo/overlays/viewsets.py

```diff
@@ -10,63 +10,60 @@
 from .models import Overlay
 
 
-class OverlayViewSet(ViewSet):
+def check_overlays_enabled():
```

Comment:
Those functions make it actually harder to read the code. Please keep those lines inline. Also, I am very keen on having only ViewSets in viewsets.py and only Views in views.py etc. I usually use utils.py for small helper functions.